### PR TITLE
Remove pandoc and tessaract installations

### PR DESCRIPTION
**Pull request by gaya3-zipstack**
## What

pandoc and tessaract-ocr is no longer a required dependency for SDK. Hence they are being removed from wherever applicable

## Why

- As they are no longer needed, they can beremoved

## How

- Remove them from dockerfiles

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

- If some unknown flow using these is not tested... A general sanity should help.

## Database Migrations

- NA

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Screenshots
Profile used
image

Indexing
image

Prompt run
image

Struture tool run
image

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: docker/docker-compose.yaml [hunk 1]
```diff
@@ -40,7 +40,7 @@ services:
     container_name: unstract-worker
     restart: unless-stopped
     entrypoint: .venv/bin/celery
-    command: "-A backend worker --loglevel=info -Q celery --autoscale=${WORKER_AUTOSCALE}"
+    command: "-A backend worker --loglevel=info -Q celery --autoscale=4,1"
```
Comment (chandrasekharan-zipstack) [inline docker/docker-compose.yaml:43]: @gaya3-zipstack any reason why this was updated?

Comment (gaya3-zipstack) [reply docker/docker-compose.yaml:43]: Hey.,,,, Thank you for catching this... Checked-in by mistake. Will revert it.

File: tools/structure/Dockerfile
Comment (chandrasekharan-zipstack) [inline tools/structure/Dockerfile:1]: @gaya3-zipstack if we're planning to publish another version of this tool, please bump the version in `config/properties.json`

**Review by chandrasekharan-zipstack — APPROVED** [review]
LGTM - please bump the structure tool version alone

**Review by muhammad-ali-e — APPROVED** [review]
LGTM, What about the liberoffice? Do we using it in structure tool?

File: tools/structure/Dockerfile
Comment (gaya3-zipstack) [reply tools/structure/Dockerfile:1]: Yes. Anyway we will need to roll versions and build for remote storage. Will be taken care there I believe.

**Pull request merged**